src/main.cpp

```diff
@@ -101,12 +105,12 @@ void runMainLoop()
             continue;
         }
       
-        if (equation.substr(0, 4) == "quit" || equation.substr(0,4) == "exit")
+        if (strcmp(equation.c_str(), "quit") == 0 || strcmp(equation.c_str(), "exit") == 0)
```

Comment:
It's generally safer to perform comparisons using std::string instead of strcmp with C strings. Would something like: `if(equation == "quit" || equation == "exit")` work? std::string has an operator== that compares against C strings.